src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -76,6 +96,7 @@ object EasyStageDataset {
 
     log.debug(s"Settings = $s")
     for {
+
```

Comment:
Actually this is probably the correct line to do the `checkFilesInBag` call. I don't remember why I moved it to line 77 later. I'll move it back.